apps/fz_vpn/lib/fz_vpn/interface.ex

```diff
@@ -0,0 +1,170 @@
+defmodule FzVpn.Interface do
+  @moduledoc """
+  This module has functions to create interfaces, set configurations on them,
+  and get peer info via [WireGuard](https://wireguard.com)
+  """
+  import FzCommon.CLI
+  import FzVpn.Interface.WGAdapter
+  import Wireguardex.DeviceConfigBuilder, except: [public_key: 2]
```

Comment:
Hmm, usually it's common to `alias` or `import` but not both. I think `alias` is usually preferred since it prevents collisions and is more explicit, but `import` seems to be used often when you're calling the imported functions heavily in current module (where `alias` would lead to lots of verbosity)